core/src/main/java/org/infinispan/stream/impl/AbstractCacheStream.java

```diff
@@ -64,24 +65,24 @@
  * @param <T> The type returned by the stream
  * @param <S> The stream interface
  */
-public abstract class AbstractCacheStream<T, S extends BaseStream<T, S>, S2 extends S> implements BaseStream<T, S> {
+public abstract class AbstractCacheStream<Original, T, S extends BaseStream<T, S>, S2 extends S> implements BaseStream<T, S> {
```

Comment:
Should we add `@param <Original>` to the javadoc?

Reply:
Done